_data/changelogs/component-range-slider.yml

```
@@ -2,6 +2,17 @@ title: Range slider
 type: component
 changelogURL:
 items:
+- date: NNNN-NN-NN
+  summary: Added optional `data-text-unit` and `data-text-preposition` attributes.
```

All the code highlighting in this entry made my eyes blur out a bit so wanted to offer a simpler suggestion
```suggestion
  summary: Added optional data attributes to customize text for screen readers.
```

---

After comparing the two, I see what you mean. Resolved in [5cc9820](https://github.com/uswds/uswds-site/pull/2310/commits/5cc9820ce3aa817dccf59c01098b620616d38969)